lib/docker/template/cache.rb

```diff
@@ -70,7 +69,7 @@ def readme(builder)
         end
 
         return unless file
-        file.safe_copy(builder.repo.cache_dir, {
+        file.safe_copy(builder.repo.cache_dir.join(file.basename), {
```

Comment:
As pointed above, the README file was being copied to the wrong place. The basename was necessary to save it to the correct path. For some reason, if you do not pass the basename with the basedir as your target, it was being copied as `repos/[reponame]/cache/[tag]` instead of `repos/[reposname]/cache/[tag]/README.md`.

Latter it would be replaced by the /tmp folder e lost.